(node:2054030) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-session-45246',
  containerIp: '10.0.3.115',
  mitmPort: 45246,
  autoAccessFixed: 1
}
2024-11-14 08:19:29.653 - [Info] MITM Version: 2
2024-11-14 08:19:29.654 - [Info] Auto Access Enabled: true
2024-11-14 08:19:29.655 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-14 08:19:29.669 - [Info] SSH man-in-the-middle server for 10.0.3.115 listening on 10.0.3.1:45246
2024-11-14 08:20:15.434 - [Debug] [Connection] Attacker connected: 218.92.0.207 | Client Identification: SSH-2.0-PuTTY
2024-11-14 08:20:16.782 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'none'
2024-11-14 08:20:16.783 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-14 08:20:17.217 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'password'
2024-11-14 08:20:17.219 - [Debug] [Auto Access] Attacker: 218.92.0.207, Threshold: 1, Attempts: 1
2024-11-14 08:20:17.219 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2024-11-14 08:20:17.223 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 1 attempts. Remaining: 5 attempts
2024-11-14 08:20:17.645 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'password'
2024-11-14 08:20:17.646 - [Debug] [Auto Access] Compromising the honeypot
2024-11-14 08:20:17.646 - [Debug] [Auto Access] Adding the following credentials: 'root:1234567890q'
2024-11-14 08:20:17.812 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-14 08:20:17.814 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-14 08:20:17.814 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.115
2024-11-14 08:20:20.709 - [Debug] [LXC] Authentication Failed
2024-11-14 08:20:20.709 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-14 08:20:20.709 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 2 attempts. Remaining: 4 attempts
2024-11-14 08:20:20.709 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-14 08:20:20.710 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 3 attempts. Remaining: 3 attempts
2024-11-14 08:20:20.713 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-14 08:20:20.714 - [Debug] [LXC] Container's OpenSSH server closed connection
2024-11-14 08:20:22.618 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'password'
2024-11-14 08:20:22.619 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 08:20:22.620 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 4 attempts. Remaining: 2 attempts
2024-11-14 08:20:24.042 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'password'
2024-11-14 08:20:24.043 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 08:20:24.051 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 5 attempts. Remaining: 1 attempts
2024-11-14 08:20:25.476 - [Debug] [Auth] Attacker 218.92.0.207 trying to authenticate with 'password'
2024-11-14 08:20:25.476 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 08:20:25.477 - [Debug] [Auth] Attacker: 218.92.0.207 has so far made 6 attempts. Remaining: 0 attempts
2024-11-14 08:20:25.477 - [Debug] [Connection] Max Login Attempts Reached - Closing connection on attacker
2024-11-14 08:20:25.478 - [Debug] [Connection] Attacker closed the connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
